### PR TITLE
Add the ability to override action for MarkdownLinkText

### PR DESCRIPTION
**Pull request by EVAST9919**

### CODE REVIEW

File: osu.Framework/Graphics/Containers/Markdown/MarkdownLinkText.cs [hunk 1]
```diff
@@ -18,39 +18,46 @@ namespace osu.Framework.Graphics.Containers.Markdown
...
                     Child = spriteText = CreateSpriteText(),
-                    Action = () => host.OpenUrlExternally(url)
+                    Action = ClickAction,
                 }
             };
 
             spriteText.Text = text;
         }
 
+        protected virtual void ClickAction() => host.OpenUrlExternally(Url);
```
Comment (smoogipoo) [inline osu.Framework/Graphics/Containers/Markdown/MarkdownLinkText.cs]: How about `OnLinkPressed()`?

File: osu.Framework/Graphics/Containers/Markdown/MarkdownLinkText.cs [hunk 2]
```diff
@@ -18,39 +18,46 @@ namespace osu.Framework.Graphics.Containers.Markdown
...
             this.text = text;
-            url = linkInline.Url ?? string.Empty;
+            Url = linkInline.Url ?? string.Empty;
 
             AutoSizeAxes = Axes.Both;
         }
 
-        [BackgroundDependencyLoader]
-        private void load(GameHost host)
+        protected override void LoadComplete()
```
Comment (smoogipoo) [inline osu.Framework/Graphics/Containers/Markdown/MarkdownLinkText.cs]: Is there a reason why you moved this to `LoadComplete()`?

Comment (EVAST9919) [reply osu.Framework/Graphics/Containers/Markdown/MarkdownLinkText.cs]: Do I need to keep it in the `load()`? I moved `GameHost` to it's own property, so I assume we don't need it

Comment (smoogipoo) [reply osu.Framework/Graphics/Containers/Markdown/MarkdownLinkText.cs]: `load()` is used to support [potential asynchronous loads](https://github.com/ppy/osu-framework/wiki/Asynchronous-loading) via `LoadComponentAsync()`, so yes.

**Pull request merged**